README.md

```diff
@@ -34,6 +34,22 @@ Allow user to create IPoIB child link and move it to the pod.
 * `master` (string, required): name of the host interface to create the link from
 * `ipam` (dictionary, required): IPAM configuration to be used for this network. For interface only without ip address, create empty dictionary, `dhcp` type is not supported.
 
+## Limitations
+
+Traffic between PODs on the same host may not work if you are using inbox driver from the Linux Kernel older than 5.8 or Mellanox OFED older than 5.1.
+
+You can apply a workaround by disabling IPoIB Enhanced mode if you need to stay on a driver version with this limitation.
+
+For inbox driver from the Kernel:
```

Comment:
nit: `For inbox driver from the Kernel:` ->  `For inbox drivers:`

Reply:
done